theory/energetic sources/supernovae.md

```diff
@@ -7,3 +7,22 @@ nav_order: 4
 ---
 
 ## Supernovae
+
```

Comment:
Please format the files before opening a PR. In VS Code, you can make use of the shortcut `Ctrl + Shift + I`